classification_analysis.ipynb

```diff
@@ -18,39 +20,31 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "2e136f3f",
    "metadata": {},
    "outputs": [],
    "source": [
     "# for reading and validating data\n",
     "import emeval.input.spec_details as eisd\n",
-    "import emeval.input.phone_view as eipv\n",
-    "import emeval.input.eval_view as eiev"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": null,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "import emeval.viz.phone_view as ezpv\n",
-    "import emeval.viz.eval_view as ezev\n",
-    "import emeval.viz.geojson as ezgj"
+    "import emeval.input.phone_view as eipv"
    ]
   },
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "ac4092a3",
    "metadata": {},
    "outputs": [],
    "source": [
     "# for analysized view\n",
-    "import emeval.analysed.phone_view as eapv"
+    "import emeval.analysed.phone_view as eapv\n",
+    "from emeval.analysed.location_smoothing import add_dist, calDistance"
```

Comment:
nit: I prefer the `import X as y` format instead of `from X import Y`
Then it makes it clear where the imported functions come from.